lib/helpers.js

```diff
@@ -24,8 +26,17 @@ const getFileType = (fileName, { transformExtensions }) => {
   return transformExtensions.test(extension) ? `${split.pop()}.${extension}` : extension;
 };
 
-const reduceAssets = (files, asset, moduleAssets) => {
-  const name = moduleAssets[asset.name] ? moduleAssets[asset.name] : asset.info.sourceFilename;
+const reduceAssets = (files, asset, moduleAssets, assetTypeModuleAssets) => {
+  console.log(assetTypeModuleAssets, asset.info.sourceFilename);
```

Comment:
This little guy snuck into the party

Reply:
Oh yea :) I'm actively debugging a last Windows issue that I can only replicate on this CI. So i'm abusing the Windows CI (I don't have access to any other Windows machine). 

I'll let you know when things are ready to look at - I thought I had it so I must be missing a tiny last detail :)